src/dmd/tokens.d

```diff
@@ -24,539 +24,292 @@ import dmd.utf;
-alias TOKdchar = TOK.TOKdchar;
-alias TOKbool = TOK.TOKbool;
-alias TOKstruct = TOK.TOKstruct;
-alias TOKclass = TOK.TOKclass;
-alias TOKinterface = TOK.TOKinterface;
-alias TOKunion = TOK.TOKunion;
-alias TOKenum = TOK.TOKenum;
-alias TOKimport = TOK.TOKimport;
-alias TOKalias = TOK.TOKalias;
-alias TOKoverride = TOK.TOKoverride;
-alias TOKdelegate = TOK.TOKdelegate;
-alias TOKfunction = TOK.TOKfunction;
-alias TOKmixin = TOK.TOKmixin;
-alias TOKalign = TOK.TOKalign;
-alias TOKextern = TOK.TOKextern;
-alias TOKprivate = TOK.TOKprivate;
-alias TOKprotected = TOK.TOKprotected;
-alias TOKpublic = TOK.TOKpublic;
-alias TOKexport = TOK.TOKexport;
-alias TOKstatic = TOK.TOKstatic;
-alias TOKfinal = TOK.TOKfinal;
-alias TOKconst = TOK.TOKconst;
-alias TOKabstract = TOK.TOKabstract;
-alias TOKdebug = TOK.TOKdebug;
-alias TOKdeprecated = TOK.TOKdeprecated;
-alias TOKin = TOK.TOKin;
-alias TOKout = TOK.TOKout;
-alias TOKinout = TOK.TOKinout;
-alias TOKlazy = TOK.TOKlazy;
-alias TOKauto = TOK.TOKauto;
-alias TOKpackage = TOK.TOKpackage;
-alias TOKmanifest = TOK.TOKmanifest;
-alias TOKimmutable = TOK.TOKimmutable;
-alias TOKif = TOK.TOKif;
-alias TOKelse = TOK.TOKelse;
-alias TOKwhile = TOK.TOKwhile;
-alias TOKfor = TOK.TOKfor;
-alias TOKdo = TOK.TOKdo;
-alias TOKswitch = TOK.TOKswitch;
-alias TOKcase = TOK.TOKcase;
-alias TOKdefault = TOK.TOKdefault;
-alias TOKbreak = TOK.TOKbreak;
-alias TOKcontinue = TOK.TOKcontinue;
-alias TOKwith = TOK.TOKwith;
-alias TOKsynchronized = TOK.TOKsynchronized;
-alias TOKreturn = TOK.TOKreturn;
-alias TOKgoto = TOK.TOKgoto;
-alias TOKtry = TOK.TOKtry;
-alias TOKcatch = TOK.TOKcatch;
-alias TOKfinally = TOK.TOKfinally;
-alias TOKasm = TOK.TOKasm;
-alias TOKforeach = TOK.TOKforeach;
-alias TOKforeach_reverse = TOK.TOKforeach_reverse;
-alias TOKscope = TOK.TOKscope;
-alias TOKon_scope_exit = TOK.TOKon_scope_exit;
-alias TOKon_scope_failure = TOK.TOKon_scope_failure;
-alias TOKon_scope_success = TOK.TOKon_scope_success;
-alias TOKinvariant = TOK.TOKinvariant;
-alias TOKunittest = TOK.TOKunittest;
-alias TOKargTypes = TOK.TOKargTypes;
-alias TOKref = TOK.TOKref;
-alias TOKmacro = TOK.TOKmacro;
-alias TOKparameters = TOK.TOKparameters;
-alias TOKtraits = TOK.TOKtraits;
-alias TOKoverloadset = TOK.TOKoverloadset;
-alias TOKpure = TOK.TOKpure;
-alias TOKnothrow = TOK.TOKnothrow;
-alias TOKgshared = TOK.TOKgshared;
-alias TOKline = TOK.TOKline;
-alias TOKfile = TOK.TOKfile;
-alias TOKfilefullpath = TOK.TOKfilefullpath;
-alias TOKmodulestring = TOK.TOKmodulestring;
-alias TOKfuncstring = TOK.TOKfuncstring;
-alias TOKprettyfunc = TOK.TOKprettyfunc;
-alias TOKshared = TOK.TOKshared;
-alias TOKat = TOK.TOKat;
-alias TOKpow = TOK.TOKpow;
-alias TOKpowass = TOK.TOKpowass;
-alias TOKgoesto = TOK.TOKgoesto;
-alias TOKvector = TOK.TOKvector;
-alias TOKpound = TOK.TOKpound;
-alias TOKinterval = TOK.TOKinterval;
-alias TOKvoidexp = TOK.TOKvoidexp;
-alias TOKcantexp = TOK.TOKcantexp;
-
-alias TOKMAX = TOK.TOKMAX;
-
-enum TOKwild = TOKinout;
```

Comment:
`TOKwild` was replaced with `TOKinout`, but I'm not 100% sure here. What was the reason for the `TOKwild` enum?